daemon/graphdriver/overlay/copy.go

```diff
@@ -105,6 +106,10 @@ func copyDir(srcDir, dstDir string, flags copyFlags) error {
 		case os.ModeNamedPipe:
 			fallthrough
 		case os.ModeSocket:
+			if rsystem.RunningInUserNS() {
```

Comment:
this seems be a typo. according to commit message, it should skip in `os.ModeDevice` case, not in `os.ModeSocket`